### PR TITLE
[hw] Use `self.logger.debug` to replace `print` to make code clean

### PR DESCRIPTION
**Pull request by csharplus**
# Description

Replace some `print` statements with `self.logger.info` for better code consistency.

# How Has This Been Tested?

- [x] Tested in a local development environment

# Checklist
- [ ] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [x] I have performed a self-review of my own code
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

cc:

@wangxiaoyou1993

### CODE REVIEW

**Comment by csharplus** [comment]
@wangxiaoyou1993 Please take a look at these minor refinement changes.

File: mage_integrations/mage_integrations/destinations/sql/base.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ def process_queries(
                         try:
                             results += self.build_connection().execute([qs], commit=True)
                         except Exception as err:
-                            print(qs)
+                            self.logger.debug(f'qs: {qs}')
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/sql/base.py:291]: can you log it with info at least?

Comment (csharplus) [reply mage_integrations/mage_integrations/destinations/sql/base.py]: Good point: made the suggested change.

File: mage_integrations/mage_integrations/destinations/trino/connectors/base.py [hunk 2]
```diff
@@ -254,7 +254,7 @@ def process_queries(
 
         if self.debug:
             for qs in query_strings:
-                print(qs, '\n')
+                self.logger.debug(f'qs: {qs}')
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/trino/connectors/base.py:257]: let's also make these info

Comment (csharplus) [reply mage_integrations/mage_integrations/destinations/trino/connectors/base.py]: Sure, made the suggested change.

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**